x-pack/plugins/cases/server/services/cases/transform.ts

```diff
@@ -192,6 +192,11 @@ export function transformSavedObjectToExternalModel(
   const status =
     STATUS_ESMODEL_TO_EXTERNAL[caseSavedObject.attributes?.status] ?? CaseStatuses.open;
 
+  // @ts-ignore
```

Comment:
I had it like that initially but it would fail if `total_alerts` and `total_comments` weren't there so instead of an ugly chain of `if`s to check the properties I picked the `delete`s.

But `?? { total_alerts: -1, total_comments: -1, } as ESCaseAttributes;` is a better solution, I'll do it 😉